docs/guides/instances.mdx

```diff
@@ -0,0 +1,112 @@
+
+<Admonition type="important">
+Qiskit Runtime v0.40.1 and later is required for the following:
+
+    - Using the instance name to specify an instance
+    - Not passing in an instance
+    - Setting a preferred instance type and region
+</Admonition>
+
+The instance's CRN is listed on the [Platform dashboard.](https://quantum.cloud.ibm.com/) Before running the below code, follow the steps to [save an account.](/docs/guides/cloud-setup#cloud-save)
+
+```python
+# Optional: Specify an instance at service level.
+# This becomes the default unless overwritten.
+service = QiskitRuntimeService(instance="<CRN>")
+backend = service.backend("<QPU-name>")
+```
+
+<span id="add-instance"></span>
+## Create an instance
+
+Follow these steps to create an instance and add it to your account. If you are an administrator who needs to set up Qiskit Runtime on Cloud for your organization, refer to [Plan Qiskit Runtime for an organization](/docs/guides/considerations-set-up-runtime).
+
+<span id="switcher"></span>
+1. To add an instance to your account, first ensure you have the correct account and region selected in the account switcher in the header.
+        * You can have (or be part of) multiple IBM Cloud accounts.  If you don't see the instances or jobs you're expecting, try a different account.
+        * You can access either region by using the same API key, but you can only see and access the instances that were created in the region that you're logged in to.
+        * If you want to create an Open instance, you must choose the us-east region.
+
+        ![The IBM Quantum Platform header is shown.  The account switcher is immediately to the right of the search bar.  The region switcher is immediately to the right of the account switcher.](/docs/images/guides/cloud-setup/IQP-Header.svg 'IBM Quantum Platform header')
+
+   <span id="regions"></span>
+   <Admonition type="note" title="About regions">
+   The instance will be created in the selected region. This region determines these important characteristics:
+     * Which QPUs are available.
+     * Where the jobs' classical computation, such as compilation, takes place.
+     * Where the user's workflow data remains.
+
+    The workflow data includes the input circuits, circuit parameters, and quantum computation results.  It does not include metadata, such as number of shots, size of circuits, or other job statistics.
+
+   To avoid exposure of the user's workflow data to a global Distributed Denial of Service protection layer, you can use [Virtual Private Endpoints](https://cloud.ibm.com/docs/overview?topic=overview-endpoints-support). `qiskit-ibm-runtime` supports this through the [private_endpoint parameter](/docs/api/qiskit-ibm-runtime/qiskit-runtime-service).
```

Comment:
```suggestion
   To avoid exposure of the user's workflow data to a global Distributed Denial of Service protection layer, you can use [Virtual Private Endpoints](https://cloud.ibm.com/docs/overview?topic=overview-endpoints-support). `qiskit-ibm-runtime` supports this through the [`private_endpoint` parameter](/docs/api/qiskit-ibm-runtime/qiskit-runtime-service).
```

Reply:
Also, should we be pointing to the recently added [VPE](https://quantum.cloud.ibm.com/docs/en/security/virtual-private-endpoints) doc?